ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplDumpTask.java

```diff
@@ -276,6 +285,9 @@ private void dumpTable(String dbName, String tblName, Path dbRoot) throws Except
           new TableExport.Paths(work.astRepresentationForErrorMsg, dbRoot, tblName, conf, true);
       String distCpDoAsUser = conf.getVar(HiveConf.ConfVars.HIVE_DISTCP_DOAS_USER);
       tuple.replicationSpec.setIsReplace(true);  // by default for all other objects this is false
+      if (AcidUtils.isTransactionalTable(tableSpec.tableHandle)) {
+        tuple.replicationSpec.setValidWriteIdList(getValidWriteIdList(dbName, tblName, validTxnList));
+      }
       new TableExport(exportPaths, tableSpec, tuple.replicationSpec, db, distCpDoAsUser, conf).write();
```

Comment:
Export ACID table have different design where they copy to temp table and export from there to avoid aborted data and is implemented as part of HIVE-18739. Current design of dumping data based on ValidWriteIdList is applicable only for replication where we have different mechanism to replicate aborted write ids at target but that is not same for import.